system/include/emscripten/wire.h

```diff
@@ -24,6 +24,7 @@
 #include <string>
 
 #define EMSCRIPTEN_ALWAYS_INLINE __attribute__((always_inline))
+#define EMSCRIPTEN_EXPORT __attribute__((visibility("default")))
```

Comment:
I'd prefer not to use `EMSCRIPTEN_EXPORT` since we may use that in the future for wasm exports. How about just `EMSCRIPTEN_VISIBILITY_DEFAULT`

Reply:
Also, how about `EMBIND_` as the prefix here?   (We can change EMSCRIPTEN_ALWAYS_INLINE too)

Reply:
Changed to EMBIND_VISIBILITY_DEFAULT, did not touch EMSCRIPTEN_ALWAYS_INLINE (yet)